102 add md calcjob (#104)
stfc/aiida-mlip: docs/source/user_guide/data.rst

.. code-block:: python

    model = ModelData.download('http://yoururl.test/model', architecture='mace', filename='model', cache_dir='/home/mlip/', force_download=False)

> -The architecture of the model file can be accessed using the `architecture` property:

Reviewer: ```suggestion
- The architecture of the model file can be accessed using the `architecture` property:
```

.. code-block:: python

    model_arch = model.architecture

> -The filepath of the model file can be accessed using the `filepath` property:

Reviewer: ```suggestion
- The filepath of the model file can be accessed using the `filepath` property:
```

.. code-block:: python

    file_path = model.filepath

.. warning::

>     When sharing data, using the ``filepath`` could point to a location inaccessible on another computer.

Reviewer: ```suggestion
    When using shared data, the ``filepath`` could point to a inaccessible location on another computer.
```

>     So if you are using data from someone else, for both the modeldata and the configfile, consider using the ``get_content()`` method to create a new file with identical content.

Reviewer: ```suggestion
    So if you are using data from someone else, for both the model data and the config file, consider using the ``get_content()`` method to create a new file with identical content.
```

    Then, use the filepath of the newly created file for running calculation.
    A more robust solution to this problem is going to be implemented.

> JanusConfigfile

Reviewer: Wondering if this documentation should be here or in the generated docs and docstrings with just a simple link to the API docs here

Author: I wonder the same, and I can (should?) add more info in the docstrings for the apidoc.
However, as a beginner/lazy user with little programming experience, I do remember finding api documentations confusing and preferring when everything was also written down in some user guide in a format with some examples.

If you are referring more specifically to the explanations for  `store_all` and `skip` that should definitely go in the api docs, maybe in a shorter format,  I put it here because it seemed a bit long.

Reviewer: Probably create an issue and get this in

---------------

The `JanusConfigfile` class is designed to handle config files written for janus-core in YAML format within the AiiDA framework.
This class inherits from `SinglefileData` in the AiiDA, and extends it to support YAML config files.
It provides methods for reading, storing, and accessing the content of the config file.

Usage
^^^^^

- To create a `JanusConfigfile` object:

.. code-block:: python

    config_file = JanusConfigfile('/path/to/config.yml')

- To read the content of the config file as a dictionary, you can use the `read_yaml()` method:

.. code-block:: python

    config_dict = config_file.read_yaml()

- To store the content of the config file in the AiiDA database, you can use the `store_content()` method:

.. code-block:: python

>     config_file.store_content(store_all: False, skip: [])

Reviewer: Are colons the right syntax here? This looks more like Groovy
